Trap Quest.inform/Inform/Extensions/Every Turn/Periodic Effects.i7x

```diff
@@ -211,15 +211,12 @@ A time based rule (this is the compute jiggles rule):
 	if a random number between 1 and 100 < 50 + unlucky:
 		if debugmode > 0, say "AUDIBLE CHECK. ";
 		let B be a random worn bra;
-		let AJC be nothing;
-		repeat with C running through worn cursed blandness clothing:
-			if C is overdress or C is bra:
-				If C is chestless or fully exposing, decide on 0;
-				If the largeness of breasts <= 2, decide on 0;
-				if B is bra and the support of B >= 2, decide on 0;
-				if (C is plentiful or C is rare) and a random number between 1 and 3 is 2, now AJC is C;
-		repeat with C running through worn AJC:
-			now AJC is C;
+		unless there is worn audible jiggles clothing or the largeness of breasts <= 2 or (B is bra and the support of B >= 2):
+			let AJC be nothing;
+			repeat with C running through worn cursed blandness ridiculously low cut or higher clothing: [ridiculously low cut or higher clothing is essentially any clothing that's not chestless or fully exposing]
+			if AJC is clothing:
+				now AJC is audible jiggles;
+					say "Your [ShortDesc of C] boinks and pings as your [BreastDesc] move! Must be some sort of magic."   
```

Comment:
Something has gone a bit wrong here. A number of mistakes.

It looks like there is a line missing after line 216. I think it was supposed to say 
```
if (C is plentiful or C is rare) and a random number between 1 and 3 is 2, now AJC is C;
```
obviously with one intendation level more than line 216.

Line 219 (the `say` command) is indented one too many times. It should have the same indentation level as the line above it (`now AJC is audible jiggles;`).


There's also one more thing I've just noticed. You've set up the RNG to trigger every time a number between 1 and 100 is less than 50. That's a huge amount, that means every 2 turns on average, the game will be looking to try and turn a piece of cursed clothing into an audible jiggles item. That's way too frequent. What would make more sense is something like `if a random number between 1 and 100 < 2 + unlucky:` because then it will be 1 in every 100 turns normally, and 1 in every 50 turns for people with the "Fashion Victim" disadvantage. This is more than enough for a random spontaneous event - in fact we may end up needing to reduce it further, or restrict what items it can happen to further. The way you have it set up now, every single cursed breast covering item without a magic-type will be becoming audible jiggles almost instantly, for the whole game!!!